### PR TITLE
Update README at cncj-202403

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the content in README.md:
- add text for CNCJ URL and description in Japanese
- add new facilitator

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
NONE

### CODE REVIEW

**Comment by atoato88** [comment]
Please not to review this PR.

I would like to want to be reveiwed by new facilitor, but who is not the approver at this folder.
I will create other PR which update the OWNERS.

**Comment by atoato88** [comment]
@shu-mutou @ziyi-xie 
Please refer the comment above. :pray:

**Comment by atoato88** [comment]
/hold
Oh, should I use this prow command in this case?

**Comment by atoato88** [comment]
@mkimuram 
Could you review this?

File: japan/cncj-202403/README.md [hunk 1]
```diff
@@ -23,6 +21,7 @@ NEED TO BE UPDATED.
 | ------------- | ------------- | ------------- | ------------- |
 |<a href="https://github.com/atoato88"><img src="https://avatars.githubusercontent.com/u/748740?s=50"></a>| <a href="https://github.com/atoato88">Akihito Inoh</a> | @akihito-inou(atoato88) | - sig-cluster-lifecycle<br> - Approver of kubernetes-sigs/kubebuilder-declarative-pattern <br> - sig-docs |
 |<a href="https://github.com/nasa9084"><img src="https://avatars.githubusercontent.com/u/11725486?s=50"></a>| <a href="https://github.com/nasa9084">nasa9084</a> | @nasa9084 | - sig-docs<br> - Approver of kubernetes/website/content/ja |
+| | <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |
```
Comment (mkimuram) [inline japan/cncj-202403/README.md:24]: @atoato88 Thank you for adding my information. Can it be updated like below to add my icon?
```
|<a href="https://github.com/mkimuram"><img src="https://avatars.githubusercontent.com/u/36450643?s=50"></a>| <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |
```

It should be looked like below:

| Icon | Name | Slack | Focus on, Role.. |
| ------------- | ------------- | ------------- | ------------- |
|<a href="https://github.com/mkimuram"></a>| <a href="https://github.com/mkimuram">Masaki Kimura</a> | @mkimura | - sig-storage |

Comment (atoato88) [reply japan/cncj-202403/README.md:24]: OK, I'll do that!

File: japan/cncj-202403/README.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
----
-
 # Kubernetes Upstream Training Japan 2024-03 by CNCJ
 
 Welcome to Kubernetes Upstream Training!!
 
 This is the location of [Kubernetes Upstream Training](https://kutj.connpass.com/event/310096)(March 13th 2024) activities.
 
+ここは、[CNCJ](https://community.cncf.io/cloud-native-community-japan/)のサブチャプターであるKubernetes Upstream Training Japan が2024/03/13に開催するトレーニングのためのページです。  
+[イベント概要と参加登録はこちら](https://kutj.connpass.com/event/310096)
```
Comment (mkimuram) [inline japan/cncj-202403/README.md:8]: @atoato88 It seems that we should add '/' at the end of connpass URL like `https://kutj.connpass.com/event/310096/`. I'm not sure if it happens in all environments, however my browser returns forbidden when I click this url without slash.

Comment (atoato88) [reply japan/cncj-202403/README.md:8]: Thank you for the precise reviewing.
I'll update this.

File: japan/cncj-202403/README.md (hunk 1 above)
Comment (atoato88) [reply japan/cncj-202403/README.md]: @mkimuram 
Done..but.. the image is big a little bit..
the `s=50` parameter does not working, I guess.

Comment (atoato88) [reply japan/cncj-202403/README.md]: Thank you for updating the suggest above!

Comment (atoato88) [reply japan/cncj-202403/README.md]: done.

File: japan/cncj-202403/README.md (hunk 2 above)
Comment (atoato88) [reply japan/cncj-202403/README.md]: done.

**Comment by atoato88** [comment]
@mkimuram 
I've done the update.
PTAL.

**Comment by mkimuram** [comment]
@atoato88 Thank you for your quick update.
/lgtm
/approve

**Pull request merged**